config/jobs/kubernetes-sigs/cluster-api/cluster-api-presubmits-release-1-3.yaml

```diff
@@ -30,17 +35,23 @@ presubmits:
     spec:
       serviceAccountName: prowjob-default-sa
       containers:
-      - command:
+      - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230727-ea685f8747-1.25
+        command:
         - runner.sh
         - ./scripts/ci-apidiff.sh
-        image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20230727-ea685f8747-1.25
+        resources:
+          requests:
+            cpu: 7300m
```

Comment:
Seems a bit excessive for apidiff :)

Reply:
IIRC there should be a dashboard somewhere to check the current usage of the jobs (not sure if that already works on the older cluster, and I don't remember where that was posted)

Reply:
There's no dashboard for the GCP clusters. Once transitioned to EKS, we'll be able to monitor on https://monitoring-eks.prow.k8s.io. I'm happy to drop apidiff down, typically we've used 2 cpu 4 Gb as a baseline for jobs, does that sound alright?

Reply:
I think that should be fine. I'm also fine with keeping it as is and then doing another iteration once we have the data